ir/type.h

```diff
@@ -229,6 +229,7 @@ class AggregateType : public Type {
     smt::expr(*map)(const IR::StateValue&, const IR::StateValue&),
     smt::expr(*reduce)(const std::set<smt::expr>&),
     const IR::StateValue &a, const IR::StateValue &b) const override;
+  StateValue aggregateVals(const std::vector<StateValue> &vals) const;
```

Comment:
seems like a tab here?

Reply:
No my emacs would never produce evil tab :), see the above lines for the indentation.